charts/komodor-agent/values.yaml

```diff
@@ -12,6 +12,11 @@ clusterName:
 # createRbac -- Creates the necessary RBAC resources for the agent - use with caution!
 createRbac: true
 
+# skipTlsVerify -- (bool) Skip TLS certificate verification for the agent (sets SKIP_TLS_VERIFY environment variable)
+skipTlsVerify: false
+# httpRequestsSkipTlsVerify -- (bool) Skip TLS certificate verification for HTTP requests (sets HTTP_REQUESTS_SKIP_TLS_VERIFY environment variable)
+httpRequestsSkipTlsVerify: false
```

Comment:
This should most likely be nested under capabilities.tasks.httpRequests ?

Reply:
ok..will move it